genai-perf/tests/test_config_command.py

```diff
@@ -73,7 +73,7 @@ def test_multi_model_name_string(self):
         user_config = yaml.safe_load(yaml_str)
         config = ConfigCommand(user_config)
 
-        self.assertEqual(config.model_names, ["gpt2_multi"])
+        self.assertEqual(config.model_names, ["gpt2", "bert"])
```

Comment:
Is there a way to test what was previously being tested here (the model name string that's going to be used/printed)?

Reply:
This functionality was moved into perf_analyzer_config.py and is tested there.

Reply:
I don't see a test there for multi-model. Can you point me to it?

Reply:
There's no specific test for multi-model - but, we do check that the non-multi-model case was unbroken whenever we check the value for the `--artifact-directory`. Didn't see a need for a specific test for this as my understanding is that the `_multi` is a GAP convention and that the artifact name doesn't actually matter (as long as we are consistent) - which is guaranteed since this is the only function that can create an artifact name.

Reply:
Right, the test was there for that consistency. Many users script the artifact ingestion, so we want to make sure the artifact naming (and anywhere else we using this naming) is consistent from release to release. It seems like a small feature unit test would be good to make sure that no changes accidentally break this.

Reply:
I've added a unit test in PA config generate to cover this case.